tdrs-backend/tdpservice/email/tasks.py

```diff
@@ -16,10 +16,35 @@
 from tdpservice.email.email import automated_email, log
 from tdpservice.email.email_enums import EmailType
 from tdpservice.parsers.util import calendar_to_fiscal
+from tdpservice.email.helpers.account_deactivation_warning import send_deactivation_email
 
 
 logger = logging.getLogger(__name__)
 
+@shared_task
+def deactivate_users():
```

Comment:
ACs say that an admin should be notified of the deactivated users. Is that happening somewhere?